addons/website_sale/static/src/js/website_sale.js

```diff
@@ -853,6 +853,7 @@ publicWidget.registry.websiteSaleProductPageReviews = publicWidget.Widget.extend
         await this._super(...arguments);
         this._updateChatterComposerPosition();
         extraMenuUpdateCallbacks.push(this._updateChatterComposerPosition.bind(this));
+        this.el.querySelector(".o_product_page_reviews_title").classList.add("collapsed");
```

Comment:
```suggestion
        this.el.querySelector('.o_product_page_reviews_title').classList.add('collapsed');
```
This scope use single quotes for technical strings in JS as well

Reply:
Applied, Thanks for the review.